streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorTopology.java

```diff
@@ -152,6 +152,24 @@ boolean isRepartitionTopic(final String topic) {
         return repartitionTopics.contains(topic);
     }
 
+    public boolean hasPersistentLocalStore() {
```

Comment:
We should add a test into `ProcessorTopologyTest` for this new method

Reply:
test case added.